### DISCUSSION TITLE
pandas

### DISCUSSION THREAD

**Discussion by gitmatsika** [flet-dev/flet#3178] [Packages] [answered]
Add pandas to flet

**Comment by FeodorFitsner** [comment, marked as answer]
[`pandas`](https://pypi.flet.dev/pandas) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.
